Assets/MixedRealityToolkit-Preview/LightEstimation-Examples/EditorGyro/EditorGyro.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+// Source code for the EditorGyro DLL can be found here:
+// https://github.com/maluoi/UnityEditorGyro
+
+// DLLs are not essential, but are nice for testing in-editor for some computers, they are currently not included
```

Comment:
this feels strange. perhaps remove this class until DLLs are distrubuted with this feature?

Reply:
It does feel strange, but the real fix for this is signing the DLL! It won't stay this way for long, so I'll leave it as is until I can get it signed properly.